src/subpackages/core/cellprofiler_core/modules/namesandtypes.py

```diff
@@ -1526,15 +1539,16 @@ def add_simple_image(
                 series,
                 index,
                 channel,
-                rescale,
+                rescale_range=rescale_range,
+                metadata_rescale=metadata_rescale,
                 volume=volume,
                 spacing=spacing,
                 z=z,
                 t=t
             )
         elif load_choice == LOAD_AS_ILLUMINATION_FUNCTION:
             provider = MonochromeImage(
-                name, url, series, index, channel, False, volume=volume, spacing=spacing, z=z, t=t
+                name, url, series, index, channel, rescale_range=None, metadata_rescale=False, volume=volume, spacing=spacing, z=z, t=t
```

Comment:
See note above about how need to ensure this keeps being treated as a special case without rescaling.